src/Ouroboros/Consensus/Ledger/Mock.hs

```diff
@@ -181,9 +181,22 @@ data SimpleBlock p c = SimpleBlock {
     }
   deriving (Generic, Show, Eq)
 
--- TODO: Write a proper condensed instance.
 instance (SimpleBlockCrypto c, OuroborosTag p) => Condense (SimpleBlock p c) where
```

Comment:
Let's add an additional constraint to `OuroborosTag` (like `Show`) so that we can call condense also on the Ouroboros payload.